server/views/post.ejs

```diff
@@ -0,0 +1,21 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Your post</title>
+    <link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
+</head>
+<body>
+    <header class="w3-container w3-margin w3-green">
+        <h2>TelePost</h2>
+    </header>
+    <div class="w3-container">
```

Comment:
really fascinating ejs file! I will have to ask for some further explanations of this as I haven't used it before.